tests/fuzz/fuzz_http2_decoder.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <qpid/dispatch/alloc_pool.h>
+
+#include "decoders/http2/http2_decoder.h"
+#include "decoders/http2/http2_test.h"
+#include "qpid/dispatch/ctools.h"
+
+#include "libFuzzingEngine.h"
+
+void qd_log_initialize(void);
+void qd_error_initialize(void);
+void qd_router_id_finalize(void);
+void qd_log_finalize(void);
+
+bool alloc_pool_initialized = false;
+
+void set_alloc_pool_initialized(bool initialized)
+{
+    alloc_pool_initialized = initialized;
+}
+
+bool is_alloc_pool_initialized(void)
+{
+    return alloc_pool_initialized;
+}
+
```

Comment:
Not a comment on this work: Having to define these functions here - I assume to avoid a linkage error (otherwise why are they defined in a header file) - implies to me that the skupper router library linked might have something a bit wrong with the factoring.